shared/js/media/image_size.js

```diff
@@ -29,7 +29,9 @@ function getImageSize(blob, callback, error) {
       error('corrupt image file');
       return;
     }
-    var magic = data.getASCIIText(0, 8);
+
+    var bytes = data.getUnsignedByteArray(0, 8);
+    var magic = String.fromCharCode.apply(String, bytes);
```

Comment:
I'd prefer it if you didn't have to make the change.  I'm okay with renaming the method, but I'd rather not change the semantics of the method.